htdocs/pages/paiement/index.php

```diff
@@ -5,7 +5,7 @@
 
 $comptaFact = new Facture($bdd);
 
-$ref = trim(mcrypt_cbc (MCRYPT_TripleDES, 'PaiementFactureAFUP_AFUP', base64_decode(str_replace(' ', '+', urldecode($_GET['ref']))), MCRYPT_DECRYPT, '@PaiFact'));
+$ref = $comptaFact->decryptLink(str_replace(' ', '+', urldecode($_GET['ref'])));
```

Comment:
Le fallback sur l'ancien chiffrement est nécessaire pour conserver le fonctionnement des liens existants

```suggestion
$ref = $comptaFact->decryptLink(str_replace(' ', '+', urldecode($_GET['ref'])));
if ($ref === false) {
    $ref = trim(mcrypt_cbc (MCRYPT_TripleDES, 'PaiementFactureAFUP_AFUP', base64_decode(str_replace(' ', '+', urldecode($_GET['ref']))), MCRYPT_DECRYPT, '@PaiFact'));
}
```